init_x: 0.0
init_y: 0.0
init_z: 1.6
hover_x: 0.5
hover_y: 0.0
hover_z: 1.17